.changeset/fix-inputWrapper.md

```diff
@@ -0,0 +1,5 @@
+---
+"react-magma-dom": patch
+---
+
+fix(inputbase): update width prop
```

Comment:
The changeset is what gets used to populate our [changelog](https://react-magma.cengage.com/version/3.2.1/api-intro/changelog/). The more descriptive we are here, the easier it is for adopters to understand what changes we introduce in each version.